### ISSUE TITLE
Interpolation in nested selectors is not supported

### ISSUE THREAD

**Issue by rusty-key** [davesnx/styled-ppx#268]
It is very useful to be able to reference another classname in a selector. For example, if your style is using class modifiers, you might want to attach them to a base class. In CSS it would look like this:
```css
.button {
  background-color: white;

  &.is-primary {
    background-color: orange;
  }
}
```

With `bs-css` it is possible it in a slightly boilerplate-y way: 
```ocaml
let buttonPrimary = [%css []]
let button = [%css [
  backgroundColor `white;
  
  selectSelf buttonPrimary [
    backgroundColor `orange;
  ]
]]

(* where selectSelf is: *)
let selectSelf = selector => Css.selector("&." ++ selector);
```

I tried to reproduce the same behavior with styled-ppx, but this fails: 
```ml
let buttonPrimary = [%cx {||}]
let button = [%cx {|
  color: white;

  &.$(buttonPrimary) {
    color: orange;
  }
```

But this fails with:
```
13 ┆ &.$(buttonPrimary) {
  14 ┆   color: orange;
  15 ┆ }

  Unexpected selector
```

Is there a way around it?

**Comment by davesnx** [comment]
This issue is fixed in latest versions, I will try to publish 0.29 with a few changes related to the integration.

**Comment by rusty-key** [comment]
❤️

**Comment by davesnx** [comment]
Published @davesnx/styled-ppx@0.29 with this fix.
